main/menu/menuMegaRenderer.c

```diff
@@ -28,7 +29,11 @@
 //==============================================================================
 
 static const paletteColor_t defaultBgColors[] = {
-    c500, c410, c320, c230, c140, c050, c041, c032, c023, c014, c005,
+    c024, c025, c035, c034, c033, c043, c143, c254, c453, c553,
+};
+// Routes through the helmet M, with three extra numbers slapped on the end so the hotdog won't crash.
```

Comment:
09f0a8dcedd6351a7663b43706cedf689dfb5f87 set `CONFIG_NUM_LEDS` to 6, and that's what gets looped to, so the three extra numbers won't really do much of anything.